### ISSUE TITLE
remove references to OpenAIRE OAI-PMH

### ISSUE THREAD

**Issue by ACz-UniBi** [openaire/guidelines-literature-repositories#30]
Dear,

it seems to be that OpenAIRE will discontinue the OAI-PMH interface in the future.

Mark as deprecated or remove references like at:

* https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/field_projectid.html

Many thanks for considering my request.
Andreas

**Comment by jdvorak001** [comment]
@ACz-UniBi, you are not suggesting OpenAIRE would no longer harvest the metadata from repositories through the OAI-PMH protocol, are you?

**Comment by ACz-UniBi** [comment]
> 
> 
> @ACz-UniBi, you are not suggesting OpenAIRE would no longer harvest the metadata from repositories through the OAI-PMH protocol, are you?

No, OpenAIRE will be discontinuing its own OAI-PMH interface in the way it is currently offered.

**Comment by jdvorak001** [comment]
Oh, I see! Thanks for the clarification.

**Comment by ACz-UniBi** [comment]
more detailed information at

https://develop.openaire.eu/farewell-oai.html
